Makefile

```diff
@@ -129,7 +129,7 @@ deploy-init:
 	$(eval export TF_VAR_paas_app_docker_image=ghcr.io/dfe-digital/find-teacher-training:$(IMAGE_TAG))
 	$(eval export TF_VAR_paas_app_config_file=./workspace_variables/app_config.yml)
 	az account set -s ${AZ_SUBSCRIPTION} && az account show
-	cd terraform && terraform init -reconfigure -backend-config=workspace_variables/$(DEPLOY_ENV)_backend.tfvars $(backend_key)
+	cd terraform && terraform init -reconfigure -upgrade -backend-config=workspace_variables/$(DEPLOY_ENV)_backend.tfvars $(backend_key)
```

Comment:
-upgrade shouldn't remain in the Makefile

Reply:
Good shout. I didn't mean to commit that one!